bucket/awsSts.json

```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/NearlyUnique/awsSts",
+    "description": "Create temporary login session for AWS CLI",
+    "version": "0.8",
+    "license": "MIT",
+    "url": "https://github.com/NearlyUnique/awsSts/releases/download/0.8/awsSts-0.8-windows.exe#/awsSts.exe",
+    "hash": "c5691ef95bd870b61ca451d176c7506cdbaf59ef9e9c534ae0c510016c7bc40a",
+    "bin": [
+        [
+            "awsSts.exe",
+            "awsSts2"
```

Comment:
Why rename it?

Reply:
The usage message for the app calls the command `awsSts2` on newer versions. I originally just had the file as `awsSts2.exe` but thought it was nicer to have the alias so both `awsSts` and `awsSts2` were available.

![image](https://user-images.githubusercontent.com/3164568/67460255-739d5800-f632-11e9-94d1-4f9dc0c3afd9.png)

Reply:
Then you can add both of them.

```
   "bin": [
        "awsSts.exe",
        [
            "awsSts.exe",
            "awsSts2"
        ]
    ],
```

Reply:
If you want both, then it have to look like this:

```suggestion
    "bin": [
        "awsSts.exe",
        [
            "awsSts.exe",
            "awsSts2"
        ]
    ],
```